src/path.rs

```diff
@@ -9,12 +9,22 @@ use std::{
 };
 
 pub const FUELUP_DIR: &str = ".fuelup";
+pub const FORC_HOME: &str = ".forc";
+pub const FUEL_HOME: &str = ".fuel";
```

Comment:
When does this get created?

Reply:
I don't think we should be removing these folders only the fuelup one. 

If I have installed forc from source then I would still want these folders present.